src/main/java/mandel/gameoflife/GridFrame.java

```diff
@@ -12,11 +12,12 @@ public class GridFrame extends JFrame {
     private boolean isPlaying;
 
     public GridFrame(Grid grid) {
-        setSize(1000, 800);
+        setSize(800, 800);
         setTitle("Game of Life");
         setDefaultCloseOperation(EXIT_ON_CLOSE);
+        int cellsize = Math.min(getHeight() / grid.getHeight(), getWidth() / grid.getWidth());
```

Comment:
What does this do?

Reply:
It creates the size of each cell based on the size of the frame and the size of the grid. It divides the height of the frame by the height of the grid and the width of the frame by the width of the grid to get the number of lines it would need to draw to fit in the frame. Then it takes the min so that if the width and height are different sizes, the cells will all fit onto the screen.